.gitignore

```diff
@@ -12,6 +12,8 @@ venv
 .coverage
 .tox/*
 
-docs/.*
-!docs/.nojekyll
+docs/.buildinfo
+docs/.doctrees
 docs/objects.inv
+
+dist
```

Comment:
what are these three additional files/folders you've added to the .gitignore?

Reply:
these are extra auto-generated files (either from publishing or from building the docs), but that we don't need to commit

Reply:
👍